app/frontend/wallet/cardano-wallet.js

```diff
@@ -162,29 +162,37 @@ const CardanoWallet = async (options) => {
     return await blockchainExplorer.fetchTxInfo(txHash)
   }
 
+  function dropUnprofitableUtxos(utxos) {
+    const profitableUtxos = []
+    const witnessSize = 139
+    let addedCost = 0
+    let inputSize = 0
+    for (const utxo of utxos) {
```

Comment:
this can be rewritten like this:
```
return utxos.filter((utxo) => {
    const inputSize = cbor.encode(TxInputFromUtxo(utxo)).length
    const addedCost = txFeeFunction(inputSize + witnessSize) - txFeeFunction(0)

    return utxo.coins > addedCost
})
```